cms/scans.py

```diff
@@ -32,7 +32,7 @@ def parser_selscan_file_conversion(parser=argparse.ArgumentParser()):
             Process a bgzipped-VCF (such as those included in the Phase 3 1000 Genomes release) into a gzip-compressed
             tped file of the sort expected by selscan. 
     """
-    parser.help = """This function converts an input VCF into a TPED file usable by selscan.
```

Comment:
not sure why this isn't displaying (i.e., for each of these subcommands) on readthedocs. Instead, it prints as follows e.g.:

Sub-commands:
selscan_file_conversion
Undocumented
usage: scans.py selscan_file_conversion ...